oauth2client/contrib/django_util/__init__.py

```diff
@@ -121,12 +136,12 @@ def optional_oauth2(request):
        if request.oauth.has_credentials():
            # this could be passed into a view
            # request.oauth.http is also initialized
-           return HttpResponse("User email: %s"
-            % request.oauth.credentials.id_token['email'])
+           return HttpResponse("User email: {0}".format(
+            request.oauth.credentials.id_token['email']))
        else:
            return HttpResponse(
-               'Here is an OAuth Authorize link: <a href="%s">Authorize</a>'
-                % request.oauth.get_authorize_redirect())
+               'Here is an OAuth Authorize link: <a href="{0}">Authorize</a>'
+                .format(request.oauth.get_authorize_redirect()))
```

Comment:
Can you break before the `.`? I'm not sure this is valid syntax. Can you verify?